requirements.txt

```diff
@@ -13,4 +13,4 @@ toposort<1.7
 torch<1.8.0; sys_platform != 'win32' and platform_machine == 'x86_64' and platform_python_implementation == 'CPython'
 typecheck-decorator<=1.2
 leabra-psyneulink<=0.3.2
-rich<=9.11.0
+rich>=9.11
```

Comment:
this should preserve the upper bound even if a new lower bound needs to be added

Reply:
Sounds good to me.